service/raft/raft_group_registry.cc

```diff
@@ -398,9 +395,13 @@ future<> raft_group_registry::abort_server(raft::group_id gid, sstring reason) {
                     rslog.warn("Failed to abort raft group server {}: {}", gid, ex);
                 });
         }
-        return s.aborted->get_future();
+        co_await s.aborted->get_future();
```

Comment:
Before the patch `direct_fd_ping` `RPC` started responding with `.group0_alive = false` as soon as we entered `server_impl::abort()`, after the patch we keep responding with `.group0_alive = true` while abort is in progress. Shouldn't we set `rg._group0_is_alive = false;` before calling `s.server->abort()`?